tests/posts_tests/test_posts_views.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+from pytest_django.asserts import assertTemplateUsed
+from django import urls
+
+
+@pytest.mark.django_db
+class TestViews:
+
+    def test_feed_view_loaded(self, client):
+        feed_response = client.get('/')
+        assert feed_response.status_code == 200
+        assert 'posts/feed.html' in feed_response.template_name
+
+    def test_about_view_loaded(self, client):
+        about_response = client.get('/about/')
+        assert about_response.status_code == 200
+        assertTemplateUsed(about_response, 'posts/about.html')
+
+    def test_resume_detail_view_redirection_and_template(self, client, persist_resume):
+        resume_detail_path = f'/post/{persist_resume.pk}/'
+        resume_detail_response = client.get(resume_detail_path)
+        assert resume_detail_response.status_code == 200
+        assert 'posts/post_detail.html' in resume_detail_response.template_name
+
+    def test_poll_detail_view_redirection_and_template(self, client, persist_poll):
+        poll_detail_path = f'/post/{persist_poll.pk}/'
+        poll_detail_response = client.get(poll_detail_path)
+        assert poll_detail_response.status_code == 200
+        assert 'posts/post_detail.html' in poll_detail_response.template_name
+
+    def test_non_existing_resume(self, client, persist_resume):
+        resume_detail_path = f'/post/{persist_resume.pk}/'
+        persist_resume.delete()
+        resume_detail_response = client.get(resume_detail_path)
+        assert resume_detail_response.status_code == 404
+
+    def test_non_existing_poll(self, client, persist_poll):
+        poll_detail_path = f'/post/{persist_poll.pk}/'
+        persist_poll.delete()
+        poll_detail_response = client.get(poll_detail_path)
+        assert poll_detail_response.status_code == 404
+
+    def test_resume_creation_view_with_authenticated_user(self, client, persist_user):
+        client.force_login(persist_user)
+        resume_creation_path = urls.reverse('resume-create')
+        resume_creation_response = client.get(resume_creation_path)
+        assert resume_creation_response.status_code == 200
+        assertTemplateUsed(resume_creation_response, 'posts/resume_form.html')
+        client.logout()
+
+    def test_resume_creation_view_without_authenticated_user(self, client):
+        resume_creation_path = urls.reverse('resume-create')
+        resume_creation_response = client.get(resume_creation_path)
+        assert resume_creation_response.status_code == 302
```

Comment:
You forgot to check the redirection template usage.

Reply:
because its 302, there is no template usage.
however I did add `resume_creation_response = client.get('/login/?next=/post/new/resume/')` for checking the response's template.